get_shas.sh

```diff
@@ -0,0 +1,15 @@
+#/bin/env bash
+
+# A script to get the shas
+version=$1
+
+if [ "$version" == "" ]; then
+  echo "Usage: get_shas.sh <version>"
+  exit 1
+fi
+
+for os in macos linux; do
```

Comment:
Seeing now that linux isn't supported platform for installing casks, we can omit looping here altogether.

Reply:
I'm doing one up, and patching the cask file while we're at it...